src/cli/init.rs

```diff
@@ -123,14 +123,12 @@ platforms = {{ platforms }}
 "#;
 
 const GITIGNORE_TEMPLATE: &str = r#"# pixi environments
-.pixi
```

Comment:
We try to design the configuration in such a way that it is machine specific and not project specific that's why we intentionally put it in the .pixi folder, because it was ignored. So I wouldn't automatically add it. Please let us know if you think there are project settings that you are setting though the config.toml.

Reply:
Alright, if this is an intentional decision, this is fine by me. I can still imagine some situations where it could be necessary to ship this file as well (for example if you know the target environment needs some mirror configuration but can’t directly control the config files on the target machines) but in these cases you can manually force this to be true.

Reply:
Yeah I see that too but this is about the default. Users can always deactivate the ignore.

Maybe print more info in the init for that reason. 🤔

Reply:
Reverted it again 👍🏻